docs/devguide/docs/swagger-docs.yaml

```diff
@@ -838,6 +838,58 @@ paths:
             application/json:
```

Comment:
we also have the openapi3.yaml file which is the one used for validations
need add that there to